CHANGELOG_PENDING.md

```diff
@@ -9,31 +9,31 @@ Friendly reminder, we have a [bug bounty program](https://hackerone.com/tendermi
 ### BREAKING CHANGES
 
 - CLI/RPC/Config
-
   - [config] \#9259 Rename the fastsync section and the fast_sync key blocksync and block_sync respectively
 
 - Apps
-
   - [abci/counter] \#6684 Delete counter example app
   - [abci] \#5783 Make length delimiter encoding consistent (`uint64`) between ABCI and P2P wire-level protocols
   - [abci] \#9145 Removes unused Response/Request `SetOption` from ABCI (@samricotta)
-  - [abci] \#8656 Added cli command for `PrepareProposal`. (@jmalicevic)
-  - [abci] \#8901 Added cli command for `ProcessProposal`. (@hvanz)
   - [abci/params] \#9287 Deduplicate `ConsensusParams` and `BlockParams` so only `types` proto definitions are used (@cmwaters)
     - Remove `TimeIotaMs` and use a hard-coded 1 millisecond value to ensure monotonically increasing block times.
     - Rename `AppVersion` to `App` so as to not stutter.
+  - [abci] \#9301 New ABCI methods `PrepareProposal` and `ProcessProposal` which give the app control over transactions proposed and allows for verification of proposed blocks.
+  - [abci] \#8656 Added cli command for `PrepareProposal`. (@jmalicevic)
+  - [abci] \#8901 Added cli command for `ProcessProposal`. (@hvanz)
```

Comment:
I'd combine these lines.

Reply:
Yes, good idea. Is it OK (in terms of current practice) to then "list" the PRs and the authors? Something like:
```md
- [abci] \#8656, \#8901 Added cli commands for `PrepareProposal`, `ProcessProposal` (@jmalicevic, @hvanz) 
```
My concern is that is might break the script that turns PR IDs into links...

Reply:
The script does seem to work on this (I tested it out locally to figure out how it works) - it's a very simple find/replace script.